src/tracks/playabletrack/wavetrack/ui/SpectrumView.h

```diff
@@ -11,9 +11,55 @@ Paul Licameli split from WaveTrackView.h
 #ifndef __AUDACITY_SPECTRUM_VIEW__
 #define __AUDACITY_SPECTRUM_VIEW__
 
+#include <unordered_map>
 #include "WaveTrackView.h" // to inherit
 
+
 class WaveTrack;
+class BrushHandle;
+using TimeFreqBinsMap = std::map<long long, std::set<wxInt64>>;
```

Comment:
Comment a bit about the purpose of this time.  It's really mapping from time (as discrete sample count) to a set of bins, right?  Then maybe (1) a better name and (2) the set need not contain wide integers, because there will not be so many frequency bin values.